### ISSUE TITLE
Improve discoverability of your work on HF

### ISSUE THREAD

**Issue by NielsRogge** [wsntxxn/AudioCaption#9]
Hi @wsntxxn,

Congrats on your work, I discovered it through the paper page: https://huggingface.co/papers/2407.14329. Niels here from the open-source team at Hugging Face.

It would be really great to link the models to the paper, see here on how to do that: https://huggingface.co/docs/hub/en/model-cards#linking-a-paper.

It's really nice to see you have created a Space: https://huggingface.co/spaces/wsntxxn/efficient_audio_captioning, however I see that the checkpoints are part of the Space. Ideally those could be pushed as separate model repositories (rather than a Space repository).

We recommend that each model checkpoint is stored in a seperate model repository, so that it can have a dedicated model card (along with tags and metadata like "audio-captioning"), as well as working download stats. See here for a guide: https://huggingface.co/docs/hub/models-uploading

Let me know if you need any help!

Cheers,

Niels
Open-source @ HF

**Comment by wsntxxn** [comment]
Hi @NielsRogge,

Sorry for the late response since I was busy last month. I have linked corresponding papers to these models: [effb2-trm-audiocaps-captioning](https://huggingface.co/wsntxxn/effb2-trm-audiocaps-captioning), [effb2-trm-clotho-captioning](https://huggingface.co/wsntxxn/effb2-trm-clotho-captioning) and [cnn14rnn-tempgru-audiocaps-captioning](https://huggingface.co/wsntxxn/cnn14rnn-tempgru-audiocaps-captioning).

Besides, I have changed the code of the Space to call the model through Hugging Face so that checkpoints are not stored in the Space repo.

Thank you very much for your help!
